### DISCUSSION TITLE
Proper Uninstallation?

### DISCUSSION THREAD

**Discussion by syrenaaa** [GooseMod/GooseMod#32] [Q&A] [answered]
Removing the update endpoint successfully removes GooseMod, but what about the downloaded themes, plugins, etc? Where can I remove them?

**Comment by valentinegb** [comment]
Plugins/Themes don't really get installed, so you don't need to worry about removing them.

**Comment by CanadaHonk** [comment, marked as answer]
Cached themes / plugins are in Chromium storage, GooseMod does not directly use the filesystem. You should be able to fully uninstall GooseMod removing the settings.json file (or just `UPDATE_ENDPOINT`s from it) and restarting via system tray.

**Reply by labunch** [reply]
could you explain that a lil better bc it still loads goosemod everytime i launch discord

**Reply by Valenplayer3000** [reply]
> could you explain that a lil better bc it still loads goosemod everytime i launch discord

When the `UPDATE_ENDPOINT` is Removed. You should need to remove the `NEW_UPDATE_ENDPOINT` 

And Restart Discord. If you see an Error just ignore this error and restart again. Now the gossemod is uninstalled completely!

**Reply by planecarplanecar** [reply]
I don't really understand what to do either.

**Reply by CanadaHonk** [reply]
> I don't really understand what to do either.

Delete your settings.json file, then restart your Discord fully (system tray). File paths:

Windows:
Stable: %appdata%\discord\settings.json
PTB:  %appdata%\discordptb\settings.json
Canary:  %appdata%\discordcanary\settings.json
Development:  %appdata%\discorddevelopment\settings.json

**Reply by planecarplanecar** [reply]
Thanks! Discord reset and became back to normal! :D
